qemu/tests/virtio_fs_sandbox.py

```diff
@@ -0,0 +1,140 @@
+import os
+import shutil
+import aexpect
+
+from avocado.utils import process
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_disk
+from virttest import utils_misc
+from virttest import utils_test
+from provider import virtio_fs_utils
+from provider import win_driver_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Virtiofs works on sandbox is none.
+     Steps:
+        1. create a common user u1.
+        2. switch to this user and create a shared dir in the home directory
+        3. start virtiofsd process with user u1.
+        3. Run virtiofsd daemons on capability shell env.
+        4. Boot a guest on the host with virtiofs options.
+        5. Log into guest then mount the virtiofs targets.
+        6. Generate files or run stress on the mount points inside guest.
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    common_user = params["common_user"]
+    error_context.context("Create an common user.", test.log.info)
+    process.system(params["del_user_cmd"], ignore_status=True, shell=True)
```

Comment:
```suggestion
```
The host environment may has a user as same as `common_user` due to some special reasons. I suggest: do NOT clear the host environment at this moment.
If test is failed due to the fact creating a user existed, we should change the value of `common_user` parameter.

Reply:
For the test, I think it's better to remove the user created in case we run it a couple of times.
And your concern about removing the existed users, it makes sense to me. I would like to create a not so common user.

Thanks.

Reply:
> For the test, I think it's better to remove the user created in case we run it a couple of times.  

Yep, that makes sense, if we create a user during case running, we should delete the user after case is finished.
However, if the user has been existed on host, we do NOT clear the host environment and we should raise a exception or something else.

>And your concern about removing the existed users, it makes sense to me. I would like to create a not so common user.
> 
> Thanks.

That's helpful to reduce the possibility of username conflict.

Reply:
Thanks. Updated, pls have a review.